### ISSUE TITLE
TextEditor.editorElement is deprecated.

### ISSUE THREAD

**Issue by seancorfield** [mauricioszabo/atom-chlorine#12]
`TextEditor.prototype.editorElement` has always been private, but now
it is gone. Reading the `editorElement` property still returns a
reference to the editor element but this field will be removed in a
later version of Atom, so we recommend using the `element` property instead.
```
TextEditor.editorElement (<embedded>:1:671452)
<unknown> (/Developer/workspace/atom-chlorine/lib/js/cljs-runtime/chlorine.aux.js:22:21)
Object.save_focus_BANG_ (/Developer/workspace/atom-chlorine/lib/js/cljs-runtime/chlorine.aux.js:24:3)
Object.conn_view (/Developer/workspace/atom-chlorine/lib/js/cljs-runtime/chlorine.ui.connection.js:56:14)
HTMLElement.chlorine$ui$connection$connect_BANG_ (/Developer/workspace/atom-chlorine/lib/js/cljs-runtime/chlorine.ui.connection.js:119:31)
CommandRegistry.handleCommandEvent (<embedded>:11:349858)
```

**Comment by seancorfield** [comment]
(auto-reported from Atom)

**Comment by seancorfield** [comment]
I think this is just a matter of changing `chlorine.aux/save-focus!` to:
```
(defn save-focus! [elem]
  (when (-> @state :last-focus nil?)
    (swap! state assoc :last-focus
           (some-> js/atom .-workspace .getActiveTextEditor .-element)))
  (js/setTimeout #(.focus (.querySelector elem "input")) 100))
```
(changing `.-editorElement` to `.-element`)

**Comment by seancorfield** [comment]
Fixed by #16 being merged. Thank you!
